executor/join_result_generators.go

```diff
@@ -34,49 +34,68 @@ var (
 // joinResultGenerator is used to generate join results according the join type, see every implementor for detailed information.
 type joinResultGenerator interface {
 	// emitMatchedInners should be called when key in outer row is equal to key in every inner row.
-	emitMatchedInners(outer Row, inners []Row, resultBuffer []Row) ([]Row, error)
+	emitMatchedInners(outer Row, inners []Row, resultBuffer []Row) ([]Row, bool, error)
```

Comment:
Need to update comment about the return bool.

Reply:
done